src/frontends/onnx/frontend/src/op/dequantize_linear.cpp

```diff
@@ -188,8 +193,78 @@ ov::OutputVector dequantize_linear(const ov::frontend::onnx::Node& node) {
     // these reshapes make sure that dequantization happens over the specified axis
     return detail::dequantize_linear(x, scale, zero_point, node.get_attribute_value<int64_t>("axis", 1), node);
 }
-ONNX_OP("DequantizeLinear", OPSET_SINCE(13), ai_onnx::opset_13::dequantize_linear);
+ONNX_OP("DequantizeLinear", {13, 18}, ai_onnx::opset_13::dequantize_linear);
 }  // namespace opset_13
+
+namespace opset_19 {
+ONNX_OP("DequantizeLinear", {19, 20}, ai_onnx::opset_13::dequantize_linear);
+}  // namespace opset_19
+
+namespace opset_21 {
+ov::OutputVector dequantize_linear(const ov::frontend::onnx::Node& node) {
+    common::default_op_checks(node, 2);
+
+    const ov::OutputVector inputs{node.get_ov_inputs()};
+    const auto& src_x = inputs[0];
+    ov::Output<ov::Node> scale = inputs[1];
+    const auto& scale_shape = scale.get_partial_shape();
+    ov::Output<ov::Node> zp;
+
+    // When no blocking dequantization is required - use regular DequantizeLinear
+    if (scale_shape.rank().is_static() && scale_shape.rank().get_length() <= 1) {
+        return ai_onnx::opset_13::dequantize_linear(node);
+    }
+
+    FRONT_END_GENERAL_CHECK(scale_shape.rank().is_static(), "Rank of the input data tensor has to be known (static).");
+    FRONT_END_GENERAL_CHECK(scale_shape.rank().get_length() == 2,
+                            "DequantizeLinear cannot operate with more than 2D scales");
+    FRONT_END_GENERAL_CHECK(src_x.get_partial_shape().is_static(),
```

Comment:
really important limitation that can be often not met. We can think of how to support it via decomposition one more time, or request new op in the opset.